src/HomeWork_2/data-access/common-transactions/delete-group.ts

```diff
@@ -0,0 +1,8 @@
+import { Groups, sequelize, UserGroups } from '../../main';
+
+export const deleteGroup = (groupId: string) => {
+  return sequelize.transaction(async () => {
+    await Groups.destroy({ where: { id: groupId } });
+    await UserGroups.destroy({ where: { groupId } });
```

Comment:
This should be done automatically on database level

> If any record gets removed from the DB, then all linked records should be removed from UserGroupas well.

`ON DELETE CASCADE` can be used to achieve this
https://www.ibm.com/docs/en/informix-servers/14.10?topic=clause-using-delete-cascade-option

Reply:
fixed